tests/libpod/render.go

```diff
@@ -20,19 +20,30 @@
 package libpod
 
 import (
+	"context"
 	"fmt"
 
+	"github.com/onsi/gomega"
```

Comment:
```suggestion
	. "github.com/onsi/gomega"
```

Import it like it is done in other files?

Reply:
Done.